supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -410,9 +410,11 @@
 ==== image:images/yes.png[yes] SSL/TLS (noun)
 *Description*: SSL/TLS refers to the Secure Socket Layer protocol (SSL) and its successor, the Transport Layer Security protocol (TLS). As both of these protocols are frequently called "SSL", always use "SSL/TLS" to avoid confusion.
 
+Use SSL/TLS in high-level documentation entries, such as headings, to establish context with encryption protocols. In other documentation areas, use TLS and document the supported version of the TLS protocol for your product.
```

Comment:
```suggestion
Use "SSL/TLS" in high-level documentation entries, such as headings, to establish context with encryption protocols. In other documentation areas, use "TLS" and document the supported version of the TLS protocol for your product.
```
Other glossary entries use quotation marks to refer to specific usage of terms.

Reply:
Hi @bburt-rh . Thanks for the review! Yes, I observed this in other entries, but quotation marks are not highly favored by the IBM SG guidelines.

IBM SG, pg. 90,: _Do not use quotation marks to emphasize words or phrases. Quotation marks can clutter the text. To emphasize words as
words, use italic font._

I'll post this to the wider CCS Style Council group and check what way is preferred for the CCS sup guide.

Reply:
@dfitzmau I agree with the IBM SG guidance on a general level, and we should probably consider changing how quotation marks are currently used in other SSG glossary entries to be consistent with the IBM guideline. 

That being said, I think that your new content should remain consistent with the current usage of quotation marks in other glossary entries, so in my opinion they need to be added here.